service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -140,6 +167,6 @@ public boolean isToBeReListed() {
 
     @JsonIgnore
     public boolean isRemote() {
-        return HearingPresence.REMOTE == presence;
+        return isNotEmpty(getAttendance()) && (getAttendance().contains(VIDEO) || getAttendance().contains(PHONE));
```

Comment:
I think it could be easier to understand this code if you extracted these two question (on either side of the "and") into their own well-named variables.

Reply:
Maybe the first one is ok as it is.

Reply:
i think it is quite easy, you cannot easily extract to variable as first check protect from NPE, 

if you do 
```
boolean isRemote.= getAttendance().contains(VIDEO) || getAttendance().contains(PHONE)
return isNotEmpty(getAttendance()) && isRemote 
```
you will get NPE if attendance is not present

Reply:
Really? This seems odd.
But that's cool if you want to leave as it is.